src/Helpers/WalletParser.cs

```diff
@@ -105,4 +108,99 @@ DerivationStrategyBase Parse(string str)
             return strategy;
         }
     }
+
+    public static string GetOutputDescriptor(Wallet wallet, string bitcoinNetwork)
```

Comment:
NIT: We could use https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods here

Reply:
done 🏁